metricbeat/docker-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
   dropwizard:
     build: ./module/dropwizard/_meta
 
-  elasticsearch:
+  elasticsearch6:
```

Comment:
I assume this needs reverting, also below.

Reply:
Everything related to this changes have been reverted since https://github.com/elastic/beats/pull/11883 was merged